CHANGES.md

```diff
@@ -3,6 +3,13 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.266
+- Update to model 0.0.198:
+  - Add cred request to api model
+  - Add AWSRegionMachineTypes endpoint to api model
+  - Update Certificates to support rosa cli in Windows
+
+
```

Comment:
This is not part of the model update, it's direct part of the SDK changes. Also, the certs are not specific to ROSA, but apply to any SDK connections on Windows :stuck_out_tongue: 
```suggestion
- Update to model 0.0.198:
  - Add cred request to api model
  - Add AWSRegionMachineTypes endpoint to api model
- windows: Update certificates
```